lib/Ohmbrewer_Temperature_Sensor.cpp

```diff
@@ -56,6 +58,7 @@ Ohmbrewer::TemperatureSensor::TemperatureSensor(int id, std::list<int>* pins, in
  * @param clonee The Equipment object to copy
  */
 Ohmbrewer::TemperatureSensor::TemperatureSensor(const TemperatureSensor& clonee) : Ohmbrewer::Equipment(clonee) {
+    _busPin = clonee.getBusPin;
```

Comment:
You're going to need to add parenthesis to this function or you're gonna have a bad time.